publish.proj

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <Import Project="$(PackagesDir)/$(FeedTasksPackage)/$(FeedTasksPackageVersion)/build/$(FeedTasksPackage).targets" />
+  <PropertyGroup>
+     <PublishPattern Condition="'$(PublishPattern)' == ''">$(PackageOutputRoot)**\*.nupkg</PublishPattern>
+     <SymbolsPackagesPattern>$(PackageOutputRoot)**\*.symbols.nupkg</SymbolsPackagesPattern>
+  </PropertyGroup>
+
+  <ItemGroup>
+     <_PackagesToPublish Include="$(PublishPattern)" Exclude="$(SymbolsPackagesPattern)">
+       <FeedUrl>$(FeedUrl)</FeedUrl>
+     </_PackagesToPublish>
+     <_PackagesToPublish Include="$(SymbolsPackagesPattern)" Condition="'$(PublishSymbols)' == 'true' and '$(SymbolsFeedUrl)' != ''">
+       <FeedUrl>$(SymbolFeedUrl)</FeedUrl>
+     </_PackagesToPublish>
+  </ItemGroup>      
+
+ <Target Name="PublishToAzureBlobFeed"
+         Inputs="%(_PackagesToPublish.FeedUrl)"
```

Comment:
While I understand this works but is it worth doing it this way instead of having unique targets for packages and symbols? This is not consistent with all the other repo's we are enabling this on.

Reply:
I don't understand the need for two separate targets.  On one of the PR's, I suggested that these be merged.

Reply:
Not a big reason beyond wanting to be able to do them individually.

Reply:
IMO, properties are a better way to control that rather than duplicating a target.

Reply:
I general I agree with you that duplication is not good but for this target it is pretty simply (2 task calls) and I think it is simpler and easier to understand as two targets. I like your changes much more then your first iteration they are just much easier to read and understand which leads to less errors.